app/managers.py

```diff
@@ -0,0 +1,48 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect(
+            "C:\\Programming Study\\Mate\\py-actor-manager\\cinema.db3"

```

Comment:
it would be easier and you could avoid using the absolute path by moving your db into the app directory

Reply:
I see, thanks!